screens/Home/MyVcs/IdInputModal.tsx

```diff
@@ -1,12 +1,18 @@
 import React from 'react';
-import {Input} from 'react-native-elements';
+import {Icon, Input, Tooltip} from 'react-native-elements';
 import {Picker} from '@react-native-picker/picker';
 import {Button, Column, Row, Text} from '../../../components/ui';
 import {Modal} from '../../../components/ui/Modal';
 import {Theme} from '../../../components/ui/styleUtils';
 import {IdInputModalProps, useIdInputModal} from './IdInputModalController';
 import {useTranslation} from 'react-i18next';
-import {I18nManager, KeyboardAvoidingView, TextInput} from 'react-native';
+import {
+  Dimensions,
+  I18nManager,
+  KeyboardAvoidingView,
+  TextInput,
+  View,
+} from 'react-native';
 import {TouchableOpacity} from 'react-native';
 import {individualId, isIOS} from '../../../shared/constants';
 import {GET_INDIVIDUAL_ID} from '../../../shared/constants';
```

Comment:
Can we merge this imports to 
`import {individualId, isIOS, GET_INDIVIDUAL_ID} from '../../../shared/constants';`

Reply:
merged the imports